frontend/components/product-list/sections/FilterableProductsSection/index.tsx

```diff
@@ -48,7 +51,12 @@ export enum ProductViewType {
 
 const PRODUCT_VIEW_TYPE_STORAGE_KEY = 'productViewType';
 
-export const FilterableProductsSection = React.memo(() => {
+type SectionProps = {
+   productList: ProductListType;
+};
+
+export const FilterableProductsSection = React.memo((props: SectionProps) => {
+   const { productList } = props;
```

Comment:
Is there a reason not to do this?
```suggestion
export const FilterableProductsSection = React.memo(({ productList }: SectionProps) => {
```

Reply:
Ha! Yes, this is the code I typed originally, but thanks to auto-formatting it added another level of indentation to the whole body of the component. This makes the code slightly harder to work with so I decided that a simple extraction line is just as easy to read and doesn't indent the body any more.